### PR TITLE
Updating minor grammar error.

### PR DESCRIPTION
**Pull request by mwmix**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind cleanup

**What this PR does / why we need it**:
Fixes a minor grammar error.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/phases/certs/certs.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func CreatePKIAssets(cfg *kubeadmapi.InitConfiguration) error {
 // CreateServiceAccountKeyAndPublicKeyFiles create a new public/private key files for signing service account users.
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/certs/certs.go]: the same typo exists in the comment here.

**Review by neolit123 — COMMENTED** [review]
thanks for the PR. i've added one minor comment.
please amend the same commit (instead of adding a second one).

thanks.

**Comment by mwmix** [comment]
Just noticed the approved marker was there only after I rebased my branch on top of master. I was attempting to get the failed tests to past but looks like that wasn't required for the merge. Sorry for any inconvenience.

**Comment by jberkus** [comment]
this merge does not require a release note

**Comment by neolit123** [comment]
@mwmix 

can you please change:
> kubeadm: Fixed minor grammatical error

to:
> NONE

**Pull request merged**